robot/src/main/java/us/ilite/robot/driverinput/DriverInput.java

```diff
@@ -120,6 +125,17 @@ private void updateDriveTrain() {
         }
     }
 
+    private void updateElevator() {
+
+        double power = 0.0d;
+
+        if (mData.driverinput.isSet(DriveTeamInputMap.DRIVER_THROTTLE_AXIS)) {
+            power = mData.driverinput.get(DriveTeamInputMap.DRIVER_THROTTLE_AXIS);
+            power *= 0.10d; //10% of the driver input.
```

Comment:
I would recommend clamping output percentage in the `Elevator` class instead of here